step1-03/demo/README.md

```diff
@@ -6,7 +6,7 @@ In this demo we are going to cover a few core basics of the language that will h
 
 ```html
 <div id="contact-form">
-  <label>Email</label><input id="email" type="email" />
+  <label for="email">Email</label><input id="email" type="email" name="email"/>
```

Comment:
Since this form won't be submitted via POST, the name property shouldn't be necessary as far as I'm aware. If you've seen guidance to the contrary, would you mind sharing a link?

Reply:
Yes, you are right. The name attribute is for server-side processing.